src/org-manager/components/organisation-details-info/organisation-details-info.component.ts

```diff
@@ -34,8 +34,10 @@ export class OrganisationDetailsInfoComponent implements OnInit {
   }
 
   public pbaNumbers(): string[] {
-    return this.org && this.org.pbaNumber && this.org.pbaNumber.length ? this.org.pbaNumber :
-      this.org && this.org.pendingPaymentAccount && this.org.pendingPaymentAccount.length ? this.org.pendingPaymentAccount : [];
+    return this.org?.pbaNumber?.length
+    ? this.org.pbaNumber
+    : this.org?.pendingPaymentAccount?.length
+      ? this.org.pendingPaymentAccount : [];
   }
```

Comment:
I am not a huge fan of all this null coalescence and double ternary. It is far too complicated to read. I would prefer this is broken down into variables and very simple return statement.